### PR TITLE
Rename: Rename Upload Text to Upload to Pastebin

### PR DESCRIPTION
**Pull request by xbdmHQ**
I accidentally deleted the fork repo not knowing it will delete the pull request lol.

Old one: https://github.com/diced/zipline/pull/378

### CODE REVIEW

**Comment by diced** [comment]
I'm unsure if this would be a good idea as it could be confused with the website, https://pastebin.com. I would be open to any other naming suggestions though.

**Comment by xSkeletor** [comment]
I agree with @diced, I think pastebin is not a good naming idea since it is directly connected to another website/service. Instead, I think leaving the current upload text format is fine and makes sense.

File: src/components/pages/Upload/Text.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export default function Text() {
     options.compression !== 'none' && req.setRequestHeader('Image-Compression-Percent', options.compression);
     options.embedded && req.setRequestHeader('Embed', 'true');
     options.zeroWidth && req.setRequestHeader('Zws', 'true');
-    options.format !== 'default' && req.setRequestHeader('Format', options.format);
+    options.format !== 'random' && req.setRequestHeader('Format', options.format);
```
Comment (TacticalTechJay) [inline src/components/pages/Upload/Text.tsx:72]: Irrelevant change. This does not seem to match with your PR's title.

**Review by TacticalTechJay — CHANGES_REQUESTED** [review]

**Comment by diced** [comment]
I will be closing this as I see the change being pretty useless, and seems like the source within the PR does not have anything to do with the PR's title.

**Pull request closed without merging**